lib/modules/datasource/git-refs/index.ts

```diff
@@ -44,7 +51,7 @@ export class GitRefsDatasource extends GitDatasource {
       releases: uniqueRefs.map((ref) => ({
         version: ref,
         gitRef: ref,
-        newDigest: rawRefs.find((rawRef) => rawRef.value === ref)?.hash,
+        newDigest: rawRefs!.find((rawRef) => rawRef.value === ref)?.hash,
```

Comment:
not required, as there is a null check before

Reply:
Unfortunately it failed the tests without it. I decided it was more elegant to add the ! than any other fix